galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 
 description = 'Galasa CICS/TS Manager'
 
-version = '0.25.0'
+version = '0.26.0'
```

Comment:
why are we incrementing the CICS Manager

Reply:
We have made an update to CicstsManagerImpl so we need to bump the version of the Manager